### PR TITLE
ES5 master with MSPointer.

### PR DESCRIPTION
**Pull request by bre1470**
Part of #18070, #18614

To-do:
- [x] Highcharts Assembler ignores eventually base option and takes regular `master`
- [x] `lint-ts` task is picky with `MSPointer`, because excluded in `tsconfig.json`

### CODE REVIEW

File: ts/tsconfig.json [hunk 1]
```diff
@@ -28,6 +28,10 @@
             "trusted-types"
         ]
     },
+    "exclude": [
+        "./Core/MSPointer.ts",
```
Comment (bre1470) [inline ts/tsconfig.json:32]: @TorsteinHonsi Should we keep MSPointer in `code/es-modules`?

Comment (TorsteinHonsi) [reply ts/tsconfig.json:32]: Yes, I guess we still need the file, just not include it in the default build?

**Comment by bre1470** [comment]
Thanks, @ppotaczek! Looks good. 👍

**Comment by TorsteinHonsi** [comment]
@bre1470 This branch sees to be based on the master, but for the new es5/es6 stuff we need to work on the `v11` branch. I tried to set a new base, but that doesn't seem to be the intension. Please advice. Should we merge the master into `v11` to get the latest Pointer compositions, then merge this one into `v11` after that?

**Review by TorsteinHonsi — APPROVED** [review]
Approved, now awaiting CI

**Pull request merged**